.github/workflows/main_flamegraph_report.yaml

```
@@ -236,30 +228,18 @@ jobs:
             echo "$HOME/.cargo/bin/inferno-collapse-perf" already found
           fi
 
-      - name: Install ethrex_l2 cli
-        run: |
-          if [ -f "$HOME/.cargo/bin/ethrex_l2" ]; then
-            echo "$HOME/.cargo/bin/ethrex_l2" already found
-          else
-            cargo install --force --path cmd/ethrex_l2
-          fi
-          ethrex_l2 config create default --default
-          ethrex_l2 config set default
-
       # By default ethrex uses revm as evm backend.
       - id: generate-flamegraph-ethrex
         name: Generate Flamegraph data for Ethrex
         shell: bash
         run: |
           rm -rf target/release/ethrex
+          cargo build --release --bin ethrex --features dev
```

I think you'll get better results building with:
```suggestion
          cargo build --profile release-with-debug --bin ethrex --features dev
```